lib/translator.js

```diff
@@ -0,0 +1,22 @@
+var get = require('lodash/get');
+var escapeRegExp = require('lodash/escapeRegExp');
+var replace = require('lodash/replace');
+var fallbackLocals = require('./translations/en');
+
+function translate(message, params, localization) {
+    if (typeof localization === 'undefined') {
+        localization = fallbackLocals;
+    }
+
+    var fallbackTransMessage = get(fallbackLocals, message, message);
+    var transMessage = get(localization, message, fallbackTransMessage);
+
+    Object.keys(params).forEach(function(key) {
```

Comment:
I'm trying to support ES5 with this codebase.
So we can probably swap `Object.keys` with some lodash helper function